CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+
```

Comment:
```suggestion
```

Reply:
This was done to run `synk`, has been reverted.